TELCODOCS-674: Document known issues for 4.11 RAN GA (#49433)
openshift/openshift-docs: release_notes/ocp-4-11-release-notes.adoc

+
(link:https://bugzilla.redhat.com/show_bug.cgi?id=2097579[*BZ#2097579*])

[id="ocp-4-11-ran-known-issues"]
== {product-title} 4.11 RAN known issues

> * After deploying a {sno} with the RAN DU profile, the `openvswitch: cpu_id mismatch with handler threads` Open vSwitch kernel logs are continuously generating. There is currently no workaround for this issue. (link:https://issues.redhat.com/browse/OCPBUGSM-46165[*OCPBUGSM-46165*])

Reviewer: ```suggestion
* After deploying a {sno} with the RAN DU profile, an error `openvswitch: cpu_id mismatch with handler threads` is continuously generated in the Open vSwitch kernel logs. There is currently no workaround for this issue. (link:https://issues.redhat.com/browse/OCPBUGSM-46165[*OCPBUGSM-46165*])
```

> * If secure boot is currently disabled and you attempt to enable it through ZTP, the installation will not begin the first time ZTP is started. When secure boot is enabled through ZTP, the boot options are configured before virtual CD is attached. Therefore, the first boot is booting into the existing HD with secure boot on. The installation then gets stuck because booting from CD is never triggered. There is currently no workaround for this issue. (link:https://issues.redhat.com/browse/OCPBUGSM-45085[*OCPBUGSM-45085*])

Reviewer: ```suggestion
* If secure boot is disabled and you attempt to enable it through ZTP, the installation does not start. When secure boot is enabled through ZTP, the boot options are configured before the virtual CD is attached. Therefore, the first boot is from the existing hard disk with secure boot turned on. The installation gets stuck because the system never boots from the CD. There is currently no workaround for this issue. (link:https://issues.redhat.com/browse/OCPBUGSM-45085[*OCPBUGSM-45085*])
```

> * If an invalid subscription channel is specified in the subscription policy used by the update, the {cgu-operator-full} indicates a successful update right after the policy is enforced because the state does not change (`AtLatestKnown`). There is currently no workaround for this issue. (link:https://issues.redhat.com/browse/OCPBUGSM-43618[*OCPBUGSM-43618*])

Reviewer: ```suggestion
* If an invalid subscription channel is specified in the subscription policy that is used to perform a cluster upgrade, the {cgu-operator-full} indicates a successful update right after the policy is enforced because the state does not change (`AtLatestKnown`). There is currently no workaround for this issue. (link:https://issues.redhat.com/browse/OCPBUGSM-43618[*OCPBUGSM-43618*])
```

> * If SRIOV-FEC policies are added separately, the {cgu-operator-full} adds the policies to the top of the policy list. The original content in the `source-crs` overrides the contents updated by the enforce policy, which causes the original inform policy to never become `Compliant`. The workaround is to add the `CatalogSource` change in the `common-config-policy`.(link:https://issues.redhat.com/browse/OCPBUGSM-39859[*OCPBUGSM-39859*])

Reviewer: {cgu-operator-full} is used in the preceding KI, no need to go again in the next KI.

```suggestion
* If SRIOV-FEC policies are added separately, the {cgu-operator} adds the policies to the top of the policy list. The original content in the `/source-crs` folder overrides the contents updated by the enforce policy, which causes the original inform policy to never become `Compliant`. The workaround is to add the `CatalogSource` change in the `common-config-policy`. (link:https://issues.redhat.com/browse/OCPBUGSM-39859[*OCPBUGSM-39859*])
```

> * The `SiteConfig` disk partition definition fails when applied to multiple nodes in a cluster. When `SiteConfig` is used to provision a compact cluster, creating a valid `diskPartition` config on multiple nodes fails with a Kustomize plugin error. There is currently no workaround for this issue. (link:https://issues.redhat.com/browse/OCPBUGSM-44403[*OCPBUGSM-44403*])

Reviewer: ```suggestion
* The `SiteConfig` disk partition definition fails when applied to multiple nodes in a cluster. When a `SiteConfig` CR is used to provision a compact cluster, creating a valid `diskPartition` config on multiple nodes fails with a Kustomize plugin error. There is currently no workaround for this issue. (link:https://issues.redhat.com/browse/OCPBUGSM-44403[*OCPBUGSM-44403*])
```

Reviewer: Wondering whether the "There is currently no workaround for this issue." statement is redundant. There are "known issues" after all. Workarounds are documented where we have them.

> * Images for `ztp-site-generate` are assigned an incorrect ArgoCD patch. When you patch the ArgoCD `Resource`, the patch points to a different `ztp-site-generate` image. There is currently no workaround for this issue. (link:https://issues.redhat.com/browse/OCPBUGSM-44261[*OCPBUGSM-44261*])

Reviewer: ```suggestion
* When you pull the `ztp-site-generate:v4.10.0-53` GitOps ZTP container and attempt to patch the ArgoCD resource, the patch points to a different ztp-site-generate image (`registry.redhat.io/openshift4/ztp-site-generate-rhel8:v4.10.0`). (link:https://issues.redhat.com/browse/OCPBUGSM-44261[*OCPBUGSM-44261*])
```

> * Installing {product-title} 4.10.13 on HPE nodes is unsuccessful. Redfish hardware cannot be inspected due to a Redfish set boot device failure. There is currently no workaround for this issue. (link:https://issues.redhat.com/browse/OCPBUGSM-44151[*OCPBUGSM-44151*])

Reviewer: ```suggestion
* Installing {product-title} 4.10.13 on Hewlett Packard Enterprise (HPE) nodes fails. Redfish hardware cannot be inspected due to a Redfish set boot device failure. (link:https://issues.redhat.com/browse/OCPBUGSM-44151[*OCPBUGSM-44151*])
```

> * Applying `BMCEventSubscription` fails to create a Redfish subscription. After the subscription YAML file is created and applied, no active Redfish subscription is visible. There is currently no workaround for this issue. (link:https://issues.redhat.com/browse/OCPBUGSM-43707[*OCPBUGSM-43707*])

Reviewer: ```suggestion
* Applying a `BMCEventSubscription` CR fails to create a Redfish events subscription for  ZT systems servers. After the subscription YAML file is created and applied, no active Redfish subscription is visible. To workaround this issue, run the following command substituting appropriate BMC parameters to call the API directly and create the subscription:
+
[source,terminal]
----
$ curl -X POST -i --insecure -u "<BMC_USERNAME>:<BMC_PASSWORD>" https://<BMC_IP>/redfish/v1/EventService/Subscriptions \
-H 'Content-Type: application/json' \
--data-raw '{
"Protocol": "Redfish",
"Context": "any string is valid",
"Destination": "https://hw-event-proxy-openshift-hw-events.apps.example.com/webhook",
"EventTypes": ["Alert"]
}'
----
+
 (link:https://issues.redhat.com/browse/OCPBUGSM-43707[*OCPBUGSM-43707*])
 ```

> * Using the ZTP process, configuration sets up two `CatalogSources` on a {sno} in a restricted environment for operators. One of the `CatalogSources` is deleted after multiple reboots. There is currently no workaround for this issue. (link:https://issues.redhat.com/browse/OCPBUGSM-46245[*OCPBUGSM-46245*])

Reviewer: ```suggestion
* Using the GitOps ZTP pipeline to install a {sno} cluter in a disconnected environment, there should be two `CatalogSource` CRs applied in the cluster. One of the `CatalogSource` CRs gets deleted following multiple node reboots. 
(link:https://issues.redhat.com/browse/OCPBUGSM-46245[*OCPBUGSM-46245*])
```

> * A number of clusters fail to update during scale testing. After starting {product-title} 4.9.26 update with Telecom vDU configuration applied, a cluster update to 4.10.13 initiated with the `ClusterVersion` CR fails to complete. There is currently no workaround for this issue. There is currently no workaround for this issue. (link:https://issues.redhat.com/browse/OCPBUGSM-44655[*OCPBUGSM-44655*])

Reviewer: ```suggestion
* A number of clusters fail to update during scale testing. After starting a {product-title} 4.9.26 update with the Telecom vDU configuration applied, a cluster update to 4.10.13 initiated with the `ClusterVersion` CR fails to complete. (link:https://issues.redhat.com/browse/OCPBUGSM-44655[*OCPBUGSM-44655*])
```

> * Spoke {sno} deployment on real bare metal (Dell PowerEdge R640) is blocked when `virtualmedia` does not disconnect the ISO after writing the image to the disk. The workaround is to manually disconnect the ISO through the virtual media tab in the `idrac` management UI. (link:https://issues.redhat.com/browse/OCPBUGSM-45884[*OCPBUGSM-45884*])

Reviewer: ```suggestion
* Using {rh-rhacm-first}, spoke cluster deployments on Dell PowerEdge R640 servers are blocked when the virtual media does not disconnect the ISO in the iDRAC console after writing the image to the disk. To workaround this issue, disconnect the ISO manually through the Virtual Media tab in the iDRAC console. (link:https://issues.redhat.com/browse/OCPBUGSM-45884[*OCPBUGSM-45884*])
```

> * Spoke bare metal host deployment using the converged workflow is blocked at inspection. An {product-title} 4.11 hub using the upstream assisted-service gets stuck in `inspecting` state, and eventually fails with an `Introspection` timeout. There is currently no workaround for this issue.(link:https://issues.redhat.com/browse/OCPBUGSM-45428[*OCPBUGSM-45428*])

Reviewer: ```suggestion
* Spoke cluster deployment on bare-metal hosts using the converged workflow is blocked at inspection. An {rh-rhacm} 4.11 hub cluster using the `assisted-service` API gets stuck in an `inspecting` state, and eventually fails with an `Introspection` timeout.
(link:https://issues.redhat.com/browse/OCPBUGSM-45428[*OCPBUGSM-45428*])
```

> * On a dual-stack environment, devices and connections get stuck on `ip-check` state with a default `dhcp6` profile generated with the `nm-initrd-generator`. The workaround is to restart the `NetworkManager` which creates the `resolve.conf` file and the installation can continue. (link:https://issues.redhat.com/browse/RHELPLAN-127788[*RHELPLAN-127788*], link:https://issues.redhat.com/browse/OCPBUGS-70[*OCPBUGS-70*])

Reviewer: ```suggestion
* On a dual-stack networking environment, devices and connections get stuck in an `ip-check` state with a default `dhcp6` profile generated by the `nm-initrd-generator` utility. The `/etc/resolve.conf` file is not generated. To workaround this issue, restart the `NetworkManager` service. This generates the missing `/etc/resolve.conf` file and the installation can continue. 
(link:https://issues.redhat.com/browse/RHELPLAN-127788[*RHELPLAN-127788*],
link:https://issues.redhat.com/browse/OCPBUGS-70[*OCPBUGS-70*])
```

> * On NVIDIA-branded NICs, incoming packets that are larger than the pre-set F5 application receive buffer (currently set to 8K) arrive with the incorrect VLAN tag. There is currently no workaround for this issue. (link:https://issues.redhat.com/browse/RHELPLAN-123058[*RHELPLAN-123058*])

Reviewer: ```suggestion
* Using NVIDIA-branded Mellanox NICs on Dell hardware, incoming packets that are larger than the preset F5 application receive buffer (currently set to 8K) arrive with an incorrect VLAN tag. This results in sporadic truncated packets being delivered by the NIC. 
(link:https://issues.redhat.com/browse/RHELPLAN-123058[*RHELPLAN-123058*])
```

> * An Operator Lifecycle Manager registry-server container fails to reach the `READY` state. When creating subscriptions with a new `CatalogSource`, the `CatalogSource` remains in a `TRANSIENT_FAILURE` state. There is currently no workaround for this issue. (link:https://issues.redhat.com/browse/OCPBUGSM-44041[*OCPBUGSM-44041*])

Reviewer: ```suggestion
* Sometimes, during ZTP GitOps pipeline {sno} installations, the Operator Lifecycle Manager registry-server container fails to reach the `READY` state. When creating subscriptions with a new `CatalogSource`, the `CatalogSource` remains in a `TRANSIENT_FAILURE` state.
(link:https://issues.redhat.com/browse/OCPBUGSM-44041[*OCPBUGSM-44041*])
```

> * A {sno} node configured with static IPv6 and deployed using ZTP becomes unreachable during Day 2 operator configuration. The {product-title} cluster installation completes successfully, and the cluster is healthy. After reboot, the node becomes unreachable with the interface down. There is currently no workaround for this issue. (link:https://issues.redhat.com/browse/OCPBUGSM-46688[*OCPBUGSM-46688*])

Reviewer: ```suggestion
* Sometimes, a {sno} node configured with static IPv6 and deployed using the GitOps ZTP pipeline becomes unreachable during Day 2 operator configuration. The {product-title} cluster installation completes successfully, and the cluster is healthy. After reboot, the node becomes unreachable with the network interface goes down. (link:https://issues.redhat.com/browse/OCPBUGSM-46688[*OCPBUGSM-46688*])
```

> * During ZTP deployment, the bare metal host image provisioning failed with the following error: `The ETag supplied did not match the ETag required to change this resource`. The workaround is to manually attach an ISO image using the Baseboard Management Controller GUI. (link:https://issues.redhat.com/browse/OCPBUGSM-46305[*OCPBUGSM-46305*])

Reviewer: ```suggestion
* During ZTP cluster deployments, the bare-metal host image provisioning fails with the following error: `The ETag supplied did not match the ETag required to change this resource`. To workaround this issue, attach the ISO image to the host using the Baseboard Management Controller GUI. 
(link:https://issues.redhat.com/browse/OCPBUGSM-46305[*OCPBUGSM-46305*])
```

> * During a master node replacement on bare metal setup, the machine is stuck in `Provisioning` state, but the node is reported as `Ready`. There is currently no workaround for this issue. (link:https://issues.redhat.com/browse/OCPBUGSM-45772[*OCPBUGSM-45772*])

Reviewer: ```suggestion
* During a master node replacement in a bare-metal cluster,  the new master host gets stuck in a `Provisioning` state, but the node reports in to the cluster as `Ready`.
(link:https://issues.redhat.com/browse/OCPBUGSM-45772[*OCPBUGSM-45772*])
```

> * In runtime, some vDU pods use the reserved cores for the platform, and {product-title} `etcd` uses the isolated application cores. There is currently no workaround for this issue. (link:https://issues.redhat.com/browse/OCPBUGSM-45078[*OCPBUGSM-45078*])

Reviewer: ```suggestion
* At runtime, when host CPU cores are reserved separately for the cluster platform and deployed applications, some vDU pods use the reserved platform cores, and the {product-title} `etcd` datastore service uses isolated application cores. 
(link:https://issues.redhat.com/browse/OCPBUGSM-45078[*OCPBUGSM-45078*])
```

+
(link:https://bugzilla.redhat.com/show_bug.cgi?id=2097579[*BZ#2097579*])

[id="ocp-4-11-ran-known-issues"]
== {product-title} 4.11 RAN known issues

* After deploying a {sno} with the RAN DU profile, an error `openvswitch: cpu_id mismatch with handler threads` is continuously generated in the Open vSwitch kernel logs. (link:https://issues.redhat.com/browse/OCPBUGSM-46165[*OCPBUGSM-46165*])

* If secure boot is currently disabled and you attempt to enable it through ZTP, the cluster installation does not start. When secure boot is enabled through ZTP, the boot options are configured before the virtual CD is attached. Therefore, the first boot is from the existing hard disk with secure boot turned on. The cluster installation gets stuck because the system never boots from the CD. Subsequent attempts to install the node through ZTP with secure boot enabled will not encounter this issue. (link:https://issues.redhat.com/browse/OCPBUGSM-45085[*OCPBUGSM-45085*])

> * If an invalid subscription channel is specified in the subscription policy that is used to perform a cluster upgrade, the {cgu-operator-full} indicates a successful upgrade right after the policy is enforced because the `Subscription` state remains. (`AtLatestKnown`). (link:https://issues.redhat.com/browse/OCPBUGSM-43618[*OCPBUGSM-43618*])

Reviewer 2: Should the '.' after "remains" be removed? The intent is to say that the state of the Subscription is still "AtLatestKnown" which prevents it from seeing that there is an update available and thus there is no upgrade performed.

Author: I accidentally added that period, updated now!

> * If the SRIOV-FEC Operator is installed through separate policies and the `CatalogSource` reuses the name of a default catalog source, the Operator installation may fail due to conflicts with the management of the default catalog sources. To avoid this issue the SRIOV-FEC `CatalogSource` CR should be added to the `common-config-policy` and the operator subscription should be added to the `common-subscriptions-policy`. As a workaround when using separate policies to install the SRIOV-FEC Operator, you must ensure that the `CatalogSource` for this Operator is uniquely named. (link:https://issues.redhat.com/browse/OCPBUGSM-39859[*OCPBUGSM-39859*])

Reviewer 2: LGTM

Reviewer 3: lgtm

* The `SiteConfig` disk partition definition fails when applied to multiple nodes in a cluster. When a `SiteConfig` CR is used to provision a compact cluster, creating a valid `diskPartition` config on multiple nodes fails with a Kustomize plugin error. (link:https://issues.redhat.com/browse/OCPBUGSM-44403[*OCPBUGSM-44403*])

> * When you patch the `ArgoCD` resource with the ZTP container, the patch points to a tag which follows the latest container version for that release version. If you want to pin the ZTP container to a specific version within the release, the patch file `argocd-openshift-gitops-patch.json` should be updated to point to the specific version. (link:https://issues.redhat.com/browse/OCPBUGSM-44261[*OCPBUGSM-44261*])

Reviewer 3: lgtm

* Installing {product-title} 4.10.13 on Hewlett Packard Enterprise (HPE) nodes fails. Redfish hardware cannot be inspected due to a Redfish set boot device failure. (link:https://issues.redhat.com/browse/OCPBUGSM-44151[*OCPBUGSM-44151*])

*  Applying a `BMCEventSubscription` CR fails to create a Redfish events subscription. After the subscription YAML file is created and applied, no active Redfish subscription is visible. To workaround this issue, run the following command substituting appropriate BMC parameters to call the API directly and create the subscription:
+
[source,terminal]
----
$ curl -X POST -i --insecure -u "<BMC_USERNAME>:<BMC_PASSWORD>" https://<BMC_IP>/redfish/v1/EventService/Subscriptions \
-H 'Content-Type: application/json' \
--data-raw '{
"Protocol": "Redfish",
"Context": "any string is valid",
"Destination": "https://hw-event-proxy-openshift-hw-events.apps.example.com/webhook",
}'
----
+

> (link:https://issues.redhat.com/browse/OCPBUGSM-43707[*OCPBUGSM-43707*])

Reviewer 4: LGTM

* During a master node replacement in a bare-metal cluster, the new master host gets stuck in a `Provisioning` state, but the node reports in to the cluster as `Ready`. (link:https://issues.redhat.com/browse/OCPBUGSM-45772[*OCPBUGSM-45772*])

* At runtime, when host CPU cores are reserved separately for the cluster platform and deployed applications, some vDU pods use the reserved platform cores, and the {product-title} `etcd` datastore service uses isolated application cores. (link:https://issues.redhat.com/browse/OCPBUGSM-45078[*OCPBUGSM-45078*])

* When you apply a tuned override to a pod and delete the tuned pod to force a restart, the pod should restart and the system should run normally. Instead a `systemd` hang can occur, causing the system to stop responding. (link:https://issues.redhat.com/browse/RHELPLAN-131021[*RHELPLAN-131021*])

> * The `dnsmasq` service causes high CPU usage after a 4.10.21 to 4.11.0-rc.1 upgrade on an SNO node. As a workaround, you can SSH into the node and restart the `dnsmasq` service by running the following command:

Reviewer 5: ```suggestion
* The `dnsmasq` service causes high CPU usage during 4.11 deployment or after 4.10 to 4.11 upgrade on an SNO node. As a workaround, you can SSH into the node and restart the `dnsmasq` service by running the following command:
```

+
[source,terminal]
----
systemctl restart dnsmasq
----
+
(link:https://issues.redhat.com/browse/OCPBUGSM-46705[*OCPBUGSM-46705*])

* When booting from a live ISO with static IPv6 address configuration, `NetworkManager` exits successfully before the interface link becomes ready, which leaves the network interface without configuration. As a workaround, edit the RHCOS ISO referenced in the `AgentServiceConfig` CR to append `rd.net.timeout.carrier` to the kernel parameters in the `grub.cfg` file:

> . Pull the `rhsoc-live` ISO image for the release being installed. The URL can be retrieved from the `AgentServiceConfig` CR on the hub cluster by running the following command:

Reviewer 5: ```suggestion
. Pull the `rhcos-live` ISO image for the release being installed. The URL can be retrieved from the `AgentServiceConfig` CR on the hub cluster by running the following command:
```

+
[source,terminal]
----
$ oc get AgentServiceConfig agent -o yaml
----

. Mount the image in the `/mnt/iso/` directory:
+
[source,terminal]
----
$ mount rhcos-live.x86_64.iso /mnt/iso/
----

. Create the `iso-grub-cfg/` directory and change to the directory:
+
[source,terminal]
----
$ mkdir iso-grub-cfg/; pushd iso-grub-cfg/
----

. Copy the contents of the `/mnt/iso/` directory to your working directory:
+
[source,terminal]
----
$ rsync -avH /mnt/iso/* .
----

> . Open the Linux boot line:

Reviewer 5: ```suggestion
. Open the GRUB configuration file:
```

+
[source,terminal]
----
$ vim EFI/redhat/grub.cfg
----

> .. Append the `rd.net.timeout.carrier=20` string to the file.

Reviewer 5: ```suggestion
.. Append the `rd.net.timeout.carrier=20` string to the linux boot line.
```

. Return to the initial working directory:
+
[source,terminal]
----
$ popd
----

. Generate the ISO file from the `iso-grub-cfg` directory:
+
[source,terminal]
----
$ mkisofs -JR -graft-points -o rhcos-carrier-timeout.iso iso-grub-cfg
----

. Push the updated ISO image to the server that is accessible by the hub cluster.

. On the hub cluster, update the `osImages` entry in the `AgentServiceConfig` CR for the installing release to point to the updated ISO image:
+
[source,terminal]
----
$ oc edit AgentServiceConfig agent
----

. Update the `url` field to point to the URL of the updated ISO image.

+
(link:https://issues.redhat.com/browse/OCPBUGSM-46336[*OCPBUGSM-46336*])

> * A kernel crash occurs after rebooting a bare-metal SNO with DU profile and workload test application. As a workaround, you can add an additional kernel parameter to the performance profile:

Reviewer 6: LGTM, on behalf of @kenyis (on behalf of Brent & Dwaine).

+
(link:https://bugzilla.redhat.com/show_bug.cgi?id=2097579[*BZ#2097579*])

[id="ocp-4-11-ran-known-issues"]
== {product-title} 4.11 RAN known issues

> * After deploying a {sno} with the RAN DU profile, an error `openvswitch: cpu_id mismatch with handler threads` is continuously generating in the Open vSwitch kernel logs. (link:https://issues.redhat.com/browse/OCPBUGSM-46165[*OCPBUGSM-46165*])

Reviewer 6: ```suggestion
* After deploying a {sno} with the RAN DU profile, an error `openvswitch: cpu_id mismatch with handler threads` is continuously generated in the Open vSwitch kernel logs. (link:https://issues.redhat.com/browse/OCPBUGSM-46165[*OCPBUGSM-46165*])
```

* If an invalid subscription channel is specified in the subscription policy that is used to perform a cluster upgrade, the {cgu-operator-full} indicates a successful upgrade right after the policy is enforced because the state does not change (`AtLatestKnown`). (link:https://issues.redhat.com/browse/OCPBUGSM-43618[*OCPBUGSM-43618*])

* If SRIOV-FEC policies are added separately, the {cgu-operator} adds the policies to the top of the policy list. The original content in the `/source-crs` folder overrides the contents updated by the enforce policy, which causes the original inform policy to never become `Compliant`. The workaround is to add the `CatalogSource` change in the `common-config-policy`. (link:https://issues.redhat.com/browse/OCPBUGSM-39859[*OCPBUGSM-39859*])

> * The `SiteConfig` disk partition definition fails when applied to multiple nodes in a cluster. When a `SiteConfig` CR is used to provision a compact cluster, creating a valid `diskPartition` config on multiple nodes fails with a Kustomize plugin error. (link:https://issues.redhat.com/browse/OCPBUGSM-44403[*OCPBUGSM-44403*])

Reviewer 2: ok

Reviewer 7: LGTM

* When you pull the `ztp-site-generate:v4.10.0-53` GitOps ZTP container and attempt to patch the ArgoCD resource, the patch points to a different ztp-site-generate image (`registry.redhat.io/openshift4/ztp-site-generate-rhel8:v4.10.0`). (link:https://issues.redhat.com/browse/OCPBUGSM-44261[*OCPBUGSM-44261*])

* Installing {product-title} 4.10.13 on Hewlett Packard Enterprise (HPE) nodes fails. Redfish hardware cannot be inspected due to a Redfish set boot device failure. (link:https://issues.redhat.com/browse/OCPBUGSM-44151[*OCPBUGSM-44151*])

> *  Applying a `BMCEventSubscription` CR fails to create a Redfish events subscription for ZT systems servers. After the subscription YAML file is created and applied, no active Redfish subscription is visible. To workaround this issue, run the following command substituting appropriate BMC parameters to call the API directly and create the subscription:

Reviewer 4: I would remove `for ZT systems servers` since this issue might not be ZT specific.

+
[source,terminal]
----
$ curl -X POST -i --insecure -u "<BMC_USERNAME>:<BMC_PASSWORD>" https://<BMC_IP>/redfish/v1/EventService/Subscriptions \
-H 'Content-Type: application/json' \
--data-raw '{
"Protocol": "Redfish",
"Context": "any string is valid",
"Destination": "https://hw-event-proxy-openshift-hw-events.apps.example.com/webhook",

> "EventTypes": ["Alert"]

Reviewer 4: Remove this line to make the workaround more general since some older ZT system does not recognize the field `EventTypes`.
```
"EventTypes": ["Alert"]
```

}'
----
+
(link:https://issues.redhat.com/browse/OCPBUGSM-43707[*OCPBUGSM-43707*])

> * Using the GitOps ZTP pipeline to install a {sno} cluter in a disconnected environment, there should be two `CatalogSource` CRs applied in the cluster. One of the `CatalogSource` CRs gets deleted following multiple node reboots. As a workaround, you can change the default names, such as `certified-operators` and `redhat-operators`, of the catalog sources. (link:https://issues.redhat.com/browse/OCPBUGSM-46245[*OCPBUGSM-46245*])

Reviewer 6: ```suggestion
* Using the GitOps ZTP pipeline to install a {sno} cluster in a disconnected environment, there should be two `CatalogSource` CRs applied in the cluster. One of the `CatalogSource` CRs gets deleted following multiple node reboots. As a workaround, you can change the default names, such as `certified-operators` and `redhat-operators`, of the catalog sources. (link:https://issues.redhat.com/browse/OCPBUGSM-46245[*OCPBUGSM-46245*])
```

> * A number of clusters fail to update during scale testing. After starting {product-title} 4.9.26 update with the Telecom vDU configuration applied, a cluster update to 4.10.13 initiated with the `ClusterVersion` CR fails to complete. (link:https://issues.redhat.com/browse/OCPBUGSM-44655[*OCPBUGSM-44655*])

Reviewer 2: ok

Reviewer 8: To be more specific,
`a cluster update to 4.10.13 initiated with the `ClusterVersion` CR fails in process 15% and waits for some cluster operators updating to target version.`

* Spoke cluster deployment on bare-metal hosts using the converged workflow is blocked at inspection. An {rh-rhacm} 4.11 hub cluster using the `assisted-service` API gets stuck in an `inspecting` state, and eventually fails with an `Introspection` timeout. (link:https://issues.redhat.com/browse/OCPBUGSM-45428[*OCPBUGSM-45428*])

* On a dual-stack networking environment, devices and connections get stuck in an `ip-check` state with a default `dhcp6` profile generated by the `nm-initrd-generator` utility. The `/etc/resolve.conf` file is not generated. To workaround this issue, restart the `NetworkManager` service. This generates the missing `/etc/resolve.conf` file and the installation can continue. (link:https://issues.redhat.com/browse/RHELPLAN-127788[*RHELPLAN-127788*], link:https://issues.redhat.com/browse/OCPBUGS-70[*OCPBUGS-70*])

* Using NVIDIA-branded Mellanox NICs on Dell hardware, incoming packets that are larger than the preset F5 application receive buffer (currently set to 8K) arrive with an incorrect VLAN tag. This results in sporadic truncated packets being delivered by the NIC. (link:https://issues.redhat.com/browse/RHELPLAN-123058[*RHELPLAN-123058*])

> * Sometimes, during ZTP GitOps pipeline {sno} installations, the Operator Lifecycle Manager registry-server container fails to reach the `READY` state. When creating subscriptions with a new `CatalogSource`, the `CatalogSource` remains in a `TRANSIENT_FAILURE` state. (link:https://issues.redhat.com/browse/OCPBUGSM-44041[*OCPBUGSM-44041*])

Reviewer 2: ok

Reviewer 9: /lgtm

Reviewer 9: /lgtm

Reviewer 9: /lgtm

+
(link:https://bugzilla.redhat.com/show_bug.cgi?id=2097579[*BZ#2097579*])

[id="ocp-4-11-ran-known-issues"]
== {product-title} 4.11 RAN known issues

> * After deploying a {sno} with the RAN DU profile, an error `openvswitch: cpu_id mismatch with handler threads`is continuously generating in the Open vSwitch kernel logs. (link:https://issues.redhat.com/browse/OCPBUGSM-46165[*OCPBUGSM-46165*])

Reviewer: ```suggestion
* After deploying a {sno} with the RAN DU profile, an error `openvswitch: cpu_id mismatch with handler threads` is continuously generating in the Open vSwitch kernel logs. (link:https://issues.redhat.com/browse/OCPBUGSM-46165[*OCPBUGSM-46165*])
```

> * If secure boot is currently disabled and you attempt to enable it through ZTP, the installation does not start. When secure boot is enabled through ZTP, the boot options are configured before the virtual CD is attached. Therefore, the first boot is from the existing hard disk with secure boot turned on. The installation gets stuck because the system never boots from the CD. (link:https://issues.redhat.com/browse/OCPBUGSM-45085[*OCPBUGSM-45085*])

Reviewer: ```suggestion
* If secure boot is currently disabled and you attempt to enable it through ZTP, the cluster installation does not start. When secure boot is enabled through ZTP, the boot options are configured before the virtual CD is attached. Therefore, the first boot is from the existing hard disk with secure boot turned on. The installation gets stuck because the system never boots from the CD. (link:https://issues.redhat.com/browse/OCPBUGSM-45085[*OCPBUGSM-45085*])
```

* During ZTP cluster deployments, the bare-metal host image provisioning fails with the following error: `The ETag supplied did not match the ETag required to change this resource`. To workaround this issue, attach the ISO image to the host using the Baseboard Management Controller GUI. (link:https://issues.redhat.com/browse/OCPBUGSM-46305[*OCPBUGSM-46305*])

* During a master node replacement in a bare-metal cluster, the new master host gets stuck in a `Provisioning` state, but the node reports in to the cluster as `Ready`. (link:https://issues.redhat.com/browse/OCPBUGSM-45772[*OCPBUGSM-45772*])

* At runtime, when host CPU cores are reserved separately for the cluster platform and deployed applications, some vDU pods use the reserved platform cores, and the {product-title} `etcd` datastore service uses isolated application cores. (link:https://issues.redhat.com/browse/OCPBUGSM-45078[*OCPBUGSM-45078*])

> * When you apply a tuned override to a pod and delete the tuned pod to force a restart, the pod should restart and the system should run normally. Instead a `systemd` hang can occur and it causes the system to stop responding. (link:https://issues.redhat.com/browse/RHELPLAN-131021[*RHELPLAN-131021*])

Reviewer 6: Prefer the following wording: "a `systemd` hang can occur, causing the system to stop responding"

Author: Updated!

+
(link:https://bugzilla.redhat.com/show_bug.cgi?id=2097579[*BZ#2097579*])

[id="ocp-4-11-ran-known-issues"]
== {product-title} 4.11 RAN known issues

> * After deploying a {sno} with the RAN DU profile, an error `openvswitch: cpu_id mismatch with handler threads` is continuously generating in the Open vSwitch kernel logs. (link:https://issues.redhat.com/browse/OCPBUGSM-46165[*OCPBUGSM-46165*])

Reviewer 6: LGTM.  Ran it by Brent, who no commented, will update later if he does.

> * If secure boot is currently disabled and you attempt to enable it through ZTP, the cluster installation does not start. When secure boot is enabled through ZTP, the boot options are configured before the virtual CD is attached. Therefore, the first boot is from the existing hard disk with secure boot turned on. The cluster installation gets stuck because the system never boots from the CD. (link:https://issues.redhat.com/browse/OCPBUGSM-45085[*OCPBUGSM-45085*])

Reviewer 2: ```suggestion
* If secure boot is currently disabled and you attempt to enable it through ZTP, the cluster installation does not start. When secure boot is enabled through ZTP, the boot options are configured before the virtual CD is attached. Therefore, the first boot is from the existing hard disk with secure boot turned on. The cluster installation gets stuck because the system never boots from the CD. Subsequent attempts to install the node via ZTP with secure boot enabled will not run into this issue. (link:https://issues.redhat.com/browse/OCPBUGSM-45085[*OCPBUGSM-45085*])
```

Reviewer 3: lgtm

> * If an invalid subscription channel is specified in the subscription policy that is used to perform a cluster upgrade, the {cgu-operator-full} indicates a successful upgrade right after the policy is enforced because the state does not change (`AtLatestKnown`). (link:https://issues.redhat.com/browse/OCPBUGSM-43618[*OCPBUGSM-43618*])

Reviewer 2: ```suggestion
* If an invalid subscription channel is specified in the subscription policy that is used to perform a cluster upgrade, the {cgu-operator-full} indicates a successful upgrade right after the policy is enforced because the Subscription state remains `AtLatestKnown`. (link:https://issues.redhat.com/browse/OCPBUGSM-43618[*OCPBUGSM-43618*])
```

> * If SRIOV-FEC policies are added separately, the {cgu-operator} adds the policies to the top of the policy list. The original content in the `/source-crs` folder overrides the contents updated by the enforce policy, which causes the original inform policy to never become `Compliant`. The workaround is to add the `CatalogSource` change in the `common-config-policy`. (link:https://issues.redhat.com/browse/OCPBUGSM-39859[*OCPBUGSM-39859*])

Reviewer 2: ```suggestion
* If installation of the SRIOV-FEC operator is done through separate policies and the CatalogSource reuses the name of a default catalog source the operator installation may fail due to conflicts with the management of the default catalog sources. To avoid this issue the SRIOV-FEC CatalogSource should be added to the `common-config-policy` and the operator subscription should be added to the `common-subscriptions-policy`. The workaround when using separate policies to install the SRIOV-FEC operator is to ensure the CatalogSource for this operator is uniquely named. (link:https://issues.redhat.com/browse/OCPBUGSM-39859[*OCPBUGSM-39859*])
```

* The `SiteConfig` disk partition definition fails when applied to multiple nodes in a cluster. When a `SiteConfig` CR is used to provision a compact cluster, creating a valid `diskPartition` config on multiple nodes fails with a Kustomize plugin error. (link:https://issues.redhat.com/browse/OCPBUGSM-44403[*OCPBUGSM-44403*])

> * When you pull the `ztp-site-generate:v4.10.0-53` GitOps ZTP container and attempt to patch the ArgoCD resource, the patch points to a different ztp-site-generate image (`registry.redhat.io/openshift4/ztp-site-generate-rhel8:v4.10.0`). (link:https://issues.redhat.com/browse/OCPBUGSM-44261[*OCPBUGSM-44261*])

Reviewer 2: When you patch the ArgoCD resource with the ZTP container the patch points to a tag which follows the latest container version for that release version. If you desire to pin the ZTP container to a specific version within the release the patch file `argocd-openshift-gitops-patch.json` should be updated to point to the specific version.

* Installing {product-title} 4.10.13 on Hewlett Packard Enterprise (HPE) nodes fails. Redfish hardware cannot be inspected due to a Redfish set boot device failure. (link:https://issues.redhat.com/browse/OCPBUGSM-44151[*OCPBUGSM-44151*])

*  Applying a `BMCEventSubscription` CR fails to create a Redfish events subscription for ZT systems servers. After the subscription YAML file is created and applied, no active Redfish subscription is visible. To workaround this issue, run the following command substituting appropriate BMC parameters to call the API directly and create the subscription:
+
[source,terminal]
----
$ curl -X POST -i --insecure -u "<BMC_USERNAME>:<BMC_PASSWORD>" https://<BMC_IP>/redfish/v1/EventService/Subscriptions \
-H 'Content-Type: application/json' \
--data-raw '{
"Protocol": "Redfish",
"Context": "any string is valid",
"Destination": "https://hw-event-proxy-openshift-hw-events.apps.example.com/webhook",
"EventTypes": ["Alert"]
}'
----
+
(link:https://issues.redhat.com/browse/OCPBUGSM-43707[*OCPBUGSM-43707*])

> * Using the GitOps ZTP pipeline to install a {sno} cluter in a disconnected environment, there should be two `CatalogSource` CRs applied in the cluster. One of the `CatalogSource` CRs gets deleted following multiple node reboots. (link:https://issues.redhat.com/browse/OCPBUGSM-46245[*OCPBUGSM-46245*])

Author: @mcornea Thank you, updated! Could you please take a look?

> * Using the GitOps ZTP pipeline to install a {sno} cluter in a disconnected environment, there should be two `CatalogSource` CRs applied in the cluster. One of the `CatalogSource` CRs gets deleted following multiple node reboots. (link:https://issues.redhat.com/browse/OCPBUGSM-46245[*OCPBUGSM-46245*])

Reviewer 5: Thanks, looks good!

* A number of clusters fail to update during scale testing. After starting {product-title} 4.9.26 update with the Telecom vDU configuration applied, a cluster update to 4.10.13 initiated with the `ClusterVersion` CR fails to complete. (link:https://issues.redhat.com/browse/OCPBUGSM-44655[*OCPBUGSM-44655*])

* Using {rh-rhacm-first}, spoke cluster deployments on Dell PowerEdge R640 servers are blocked when the virtual media does not disconnect the ISO in the iDRAC console after writing the image to the disk. To workaround this issue, disconnect the ISO manually through the Virtual Media tab in the iDRAC console. (link:https://issues.redhat.com/browse/OCPBUGSM-45884[*OCPBUGSM-45884*])

> * Spoke cluster deployment on bare-metal hosts using the converged workflow is blocked at inspection. An {rh-rhacm} 4.11 hub cluster using the `assisted-service` API gets stuck in an `inspecting` state, and eventually fails with an `Introspection` timeout. (link:https://issues.redhat.com/browse/OCPBUGSM-45428[*OCPBUGSM-45428*])

Reviewer 6: LGTM (according to Tao, via email).

* During ZTP cluster deployments, the bare-metal host image provisioning fails with the following error: `The ETag supplied did not match the ETag required to change this resource`. To workaround this issue, attach the ISO image to the host using the Baseboard Management Controller GUI. (link:https://issues.redhat.com/browse/OCPBUGSM-46305[*OCPBUGSM-46305*])

* During a master node replacement in a bare-metal cluster, the new master host gets stuck in a `Provisioning` state, but the node reports in to the cluster as `Ready`. (link:https://issues.redhat.com/browse/OCPBUGSM-45772[*OCPBUGSM-45772*])

* At runtime, when host CPU cores are reserved separately for the cluster platform and deployed applications, some vDU pods use the reserved platform cores, and the {product-title} `etcd` datastore service uses isolated application cores. (link:https://issues.redhat.com/browse/OCPBUGSM-45078[*OCPBUGSM-45078*])

> * The log collector pod has higher CPU usage than the same pod in previous versions when you deploy a {sno} with the vRAN DU profile. (link:https://issues.redhat.com/browse/OCPBUGSM-46986[*OCPBUGSM-46986*])

Reviewer 6: This RN should be removed, further analysis showed that the issue was resolved in 4.11, see https://issues.redhat.com/browse/LOG-2958.

> * When you apply a tuned override to a pod and delete the tuned pod to force a restart, the pod should restart and the system should run normally. Instead a `systemd` hang occurs and causes the system to stop responding. (link:https://issues.redhat.com/browse/RHELPLAN-131021[*RHELPLAN-131021*])

Reviewer 6: Should say that a hang *may* occur and cause the system to stop responding (as discussed with Dwaine G that this issue was only seen one time).

+
(link:https://bugzilla.redhat.com/show_bug.cgi?id=2097579[*BZ#2097579*])

[id="ocp-4-11-ran-known-issues"]
== {product-title} 4.11 RAN known issues

* After deploying a {sno} with the RAN DU profile, an error `openvswitch: cpu_id mismatch with handler threads` is continuously generated in the Open vSwitch kernel logs. (link:https://issues.redhat.com/browse/OCPBUGSM-46165[*OCPBUGSM-46165*])

> * If secure boot is currently disabled and you attempt to enable it through ZTP, the cluster installation does not start. When secure boot is enabled through ZTP, the boot options are configured before the virtual CD is attached. Therefore, the first boot is from the existing hard disk with secure boot turned on. The cluster installation gets stuck because the system never boots from the CD. (link:https://issues.redhat.com/browse/OCPBUGSM-45085[*OCPBUGSM-45085*])

Reviewer 10: ```suggestion
* If secure boot is currently disabled and you try to enable it using ZTP, the cluster installation does not start. When secure boot is enabled through ZTP, the boot options are configured before the virtual CD is attached. Therefore, the first boot is from the existing hard disk with secure boot turned on. The cluster installation gets stuck because the system never boots from the CD. (link:https://issues.redhat.com/browse/OCPBUGSM-45085[*OCPBUGSM-45085*])
```

Reviewer 10: This suggestion uses 'Red Hat simple words' stye.

* If an invalid subscription channel is specified in the subscription policy that is used to perform a cluster upgrade, the {cgu-operator-full} indicates a successful upgrade right after the policy is enforced because the `Subscription` state remains `AtLatestKnown`. (link:https://issues.redhat.com/browse/OCPBUGSM-43618[*OCPBUGSM-43618*])

> * If the SRIOV-FEC Operator is installed through separate policies and the `CatalogSource` reuses the name of a default catalog source, the Operator installation may fail due to conflicts with the management of the default catalog sources. To avoid this issue the SRIOV-FEC `CatalogSource` CR should be added to the `common-config-policy` and the operator subscription should be added to the `common-subscriptions-policy`. As a workaround when using separate policies to install the SRIOV-FEC Operator, you must ensure that the `CatalogSource` for this Operator is uniquely named. (link:https://issues.redhat.com/browse/OCPBUGSM-39859[*OCPBUGSM-39859*])

Reviewer 10: ```suggestion
* If the SRIOV-FEC Operator is installed through separate policies and the `CatalogSource` reuses the name of a default catalog source, the Operator installation might fail due to conflicts with the management of the default catalog sources. To avoid this issue the SRIOV-FEC `CatalogSource` CR should be added to the `common-config-policy` and the operator subscription should be added to the `common-subscriptions-policy`. As a workaround when using separate policies to install the SRIOV-FEC Operator, you must ensure that the `CatalogSource` for this Operator is uniquely named. (link:https://issues.redhat.com/browse/OCPBUGSM-39859[*OCPBUGSM-39859*])
```

Reviewer 10: See the ISG entry for 'may': https://www.ibm.com/docs/en/ibm-style?topic=word-usage

> * The `SiteConfig` disk partition definition fails when applied to multiple nodes in a cluster. When a `SiteConfig` CR is used to provision a compact cluster, creating a valid `diskPartition` config on multiple nodes fails with a Kustomize plugin error. (link:https://issues.redhat.com/browse/OCPBUGSM-44403[*OCPBUGSM-44403*])

Reviewer 10: ```suggestion
* The `SiteConfig` disk partition definition fails when applied to multiple nodes in a cluster. When a `SiteConfig` CR is used to provision a compact cluster, creating a valid `diskPartition` config on multiple nodes fails with a Kustomize plugin error. (link:https://issues.redhat.com/browse/OCPBUGSM-44403[*OCPBUGSM-44403*])
```
```suggestion
* The `SiteConfig` disk partition definition fails when applied to multiple nodes in a cluster. When a `SiteConfig` CR is used to provision a compact cluster, creating a valid `diskPartition` config on multiple nodes fails with a Kustomize plug-in error. (link:https://issues.redhat.com/browse/OCPBUGSM-44403[*OCPBUGSM-44403*])
```

Reviewer 10: See the ISG for information on 'plugin': https://www.ibm.com/docs/en/ibm-style?topic=word-usage

*  Applying a `BMCEventSubscription` CR fails to create a Redfish events subscription. After the subscription YAML file is created and applied, no active Redfish subscription is visible. To workaround this issue, run the following command substituting appropriate BMC parameters to call the API directly and create the subscription:
+
[source,terminal]
----
$ curl -X POST -i --insecure -u "<BMC_USERNAME>:<BMC_PASSWORD>" https://<BMC_IP>/redfish/v1/EventService/Subscriptions \
-H 'Content-Type: application/json' \
--data-raw '{
"Protocol": "Redfish",
"Context": "any string is valid",
"Destination": "https://hw-event-proxy-openshift-hw-events.apps.example.com/webhook",
}'
----
+
(link:https://issues.redhat.com/browse/OCPBUGSM-43707[*OCPBUGSM-43707*])

* Using the GitOps ZTP pipeline to install a {sno} cluster in a disconnected environment, there should be two `CatalogSource` CRs applied in the cluster. One of the `CatalogSource` CRs gets deleted following multiple node reboots. As a workaround, you can change the default names, such as `certified-operators` and `redhat-operators`, of the catalog sources. (link:https://issues.redhat.com/browse/OCPBUGSM-46245[*OCPBUGSM-46245*])

> * A number of clusters fail to update during scale testing. After starting {product-title} 4.9.26 update with the Telecom vDU configuration applied, a cluster update to 4.10.13 initiated with the `ClusterVersion` CR fails at 15% and waits for cluster Operators updating to the target version. (link:https://issues.redhat.com/browse/OCPBUGSM-44655[*OCPBUGSM-44655*])

Reviewer 10: ```suggestion
* Several clusters fail to update during scale testing. After starting {product-title} 4.9.26 update with the Telecom vDU configuration applied, a cluster update to 4.10.13 initiated with the `ClusterVersion` CR fails at 15% and waits for cluster Operators updating to the target version. (link:https://issues.redhat.com/browse/OCPBUGSM-44655[*OCPBUGSM-44655*])
```

* Spoke cluster deployment on bare-metal hosts using the converged workflow is blocked at inspection. An {rh-rhacm} 4.11 hub cluster using the `assisted-service` API gets stuck in an `inspecting` state, and eventually fails with an `Introspection` timeout. (link:https://issues.redhat.com/browse/OCPBUGSM-45428[*OCPBUGSM-45428*])

* Sometimes, during ZTP GitOps pipeline {sno} installations, the Operator Lifecycle Manager registry-server container fails to reach the `READY` state. When creating subscriptions with a new `CatalogSource`, the `CatalogSource` remains in a `TRANSIENT_FAILURE` state. (link:https://issues.redhat.com/browse/OCPBUGSM-44041[*OCPBUGSM-44041*])

* When you apply a tuned override to a pod and delete the tuned pod to force a restart, the pod should restart and the system should run normally. Instead a `systemd` hang can occur, causing the system to stop responding. (link:https://issues.redhat.com/browse/RHELPLAN-131021[*RHELPLAN-131021*])

> * When booting ZT system machines from a live ISO with static IPv6 address configuration, `NetworkManager` exits successfully before the interface link becomes ready, which leaves the network interface without configuration. As a workaround, edit the RHCOS ISO referenced in the `AgentServiceConfig` CR to append `rd.net.timeout.carrier` to the kernel parameters in the `grub.cfg` file:

Reviewer 10: ```suggestion
* When booting ZTP system machines from a live ISO with static IPv6 address configuration, `NetworkManager` exits successfully before the interface link becomes ready, which leaves the network interface without configuration. As a workaround, edit the RHCOS ISO referenced in the `AgentServiceConfig` CR to append `rd.net.timeout.carrier` to the kernel parameters in the `grub.cfg` file:
```

Reviewer 10: I think you meant 'ZTP' here, yes?

Author: @tmalove Thanks for spotting this! ZT Systems is actually a computer manufacturer and some customers use their hardware and we're referring to that. I'll update it to `ZT Systems machines`

+
[source,terminal]
----
$ mount rhcos-live.x86_64.iso /mnt/iso/
----

. Create the `iso-grub-cfg/` directory and change to the directory:
+
[source,terminal]
----
$ mkdir iso-grub-cfg/; pushd iso-grub-cfg/
----

. Copy the contents of the `/mnt/iso/` directory to your working directory:
+
[source,terminal]
----
$ rsync -avH /mnt/iso/* .
----

. Open the GRUB configuration file:
+
[source,terminal]
----
$ vim EFI/redhat/grub.cfg
----

.. Append the `rd.net.timeout.carrier=20` string to the `linux` boot line.

. Return to the initial working directory:
+
[source,terminal]
----
$ popd
----

. Generate the ISO file from the `iso-grub-cfg` directory:
+
[source,terminal]
----
$ mkisofs -JR -graft-points -o rhcos-carrier-timeout.iso iso-grub-cfg
----

. Push the updated ISO image to the server that is accessible by the hub cluster.

. On the hub cluster, update the `osImages` entry in the `AgentServiceConfig` CR for the installing release to point to the updated ISO image:
+
[source,terminal]
----
$ oc edit AgentServiceConfig agent
----

. Update the `url` field to point to the URL of the updated ISO image.

+
(link:https://issues.redhat.com/browse/OCPBUGSM-46336[*OCPBUGSM-46336*])

> * A kernel crash occurs after rebooting a bare-metal SNO with DU profile and workload test application. As a workaround, you can add an additional kernel parameter to the performance profile:

Reviewer 10: ```suggestion
* A kernel failure occurs after rebooting a bare-metal SNO with DU profile and workload test application. As a workaround, you can add an additional kernel parameter to the performance profile:
```

Reviewer 10: See the ISG entry for 'crash': https://www.ibm.com/docs/en/ibm-style?topic=word-usage

+
[source,yaml]
----
apiVersion: performance.openshift.io/v2
kind: PerformanceProfile
spec:
  additionalKernelArgs:
  - rcutree.kthread_prio=11
----
+
(link:https://issues.redhat.com/browse/RHELPLAN-123262[*RHELPLAN-123262*])

* During ZTP cluster deployments, the bare-metal host image provisioning fails with the following error: `The ETag supplied did not match the ETag required to change this resource`. To workaround this issue, attach the ISO image to the host using the Baseboard Management Controller GUI. (link:https://issues.redhat.com/browse/OCPBUGSM-46305[*OCPBUGSM-46305*])

> * During a master node replacement in a bare-metal cluster, the new master host gets stuck in a `Provisioning` state, but the node reports in to the cluster as `Ready`. (link:https://issues.redhat.com/browse/OCPBUGSM-45772[*OCPBUGSM-45772*])

Reviewer 10: See the ISG entry for 'master': https://www.ibm.com/docs/en/ibm-style?topic=word-usage

> * At runtime, when host CPU cores are reserved separately for the cluster platform and deployed applications, some vDU pods use the reserved platform cores, and the {product-title} `etcd` datastore service uses isolated application cores. (link:https://issues.redhat.com/browse/OCPBUGSM-45078[*OCPBUGSM-45078*])

Reviewer 10: ```suggestion
* At runtime, when host CPU cores are reserved separately for the cluster platform and deployed applications, some vDU pods use the reserved platform cores, and the {product-title} `etcd` data store service uses isolated application cores. (link:https://issues.redhat.com/browse/OCPBUGSM-45078[*OCPBUGSM-45078*])
```

Reviewer 10: See the ISG entry for 'datastore': https://www.ibm.com/docs/en/ibm-style?topic=word-usage

$ curl -X POST -i --insecure -u "<BMC_USERNAME>:<BMC_PASSWORD>" https://<BMC_IP>/redfish/v1/EventService/Subscriptions \
-H 'Content-Type: application/json' \
--data-raw '{
"Protocol": "Redfish",
"Context": "any string is valid",
"Destination": "https://hw-event-proxy-openshift-hw-events.apps.example.com/webhook",
}'
----
+
(link:https://issues.redhat.com/browse/OCPBUGSM-43707[*OCPBUGSM-43707*])

* Using the GitOps ZTP pipeline to install a {sno} cluster in a disconnected environment, there should be two `CatalogSource` CRs applied in the cluster. One of the `CatalogSource` CRs gets deleted following multiple node reboots. As a workaround, you can change the default names, such as `certified-operators` and `redhat-operators`, of the catalog sources. (link:https://issues.redhat.com/browse/OCPBUGSM-46245[*OCPBUGSM-46245*])

* Several clusters fail to update during scale testing. After starting {product-title} 4.9.26 update with the Telecom vDU configuration applied, a cluster update to 4.10.13 initiated with the `ClusterVersion` CR fails at 15% and waits for cluster Operators updating to the target version. (link:https://issues.redhat.com/browse/OCPBUGSM-44655[*OCPBUGSM-44655*])

> * Spoke cluster deployment on bare-metal hosts using the converged workflow is blocked at inspection. An {rh-rhacm} 4.11 hub cluster using the `assisted-service` API gets stuck in an `inspecting` state, and eventually fails with an `Introspection` timeout. (link:https://issues.redhat.com/browse/OCPBUGSM-45428[*OCPBUGSM-45428*])

Reviewer 11: OCPBUGSM-45428 was already fixed in bz https://bugzilla.redhat.com/show_bug.cgi?id=2096460 - I'm not sure why the OCP bug is still open but I will verify it. This entry can be removed

* Sometimes, a {sno} node configured with static IP and deployed using the GitOps ZTP pipeline becomes unreachable during Day 2 operator configuration. The {product-title} cluster installation completes successfully, and the cluster is healthy. After reboot, the node becomes unreachable with the network interface down. (link:https://issues.redhat.com/browse/OCPBUGSM-46688[*OCPBUGSM-46688*])

* After removing the `SriovNetworkNodePolicy` policy from the Git repository, the `SriovNetworkNodePolicy` resource managed by the removed policy remains on the spoke cluster. (link:https://issues.redhat.com/browse/OCPBUGSM-34614[*OCPBUGSM-34614*])

* When upgrading the PTP Operator from 4.10 to 4.11, the OpenShift subscription reports the error 'no channel heads (entries not replaced by another entry) found in channel "stable" of package "ptp-operator"'. However, the Operator upgrades successfully. (link:https://issues.redhat.com/browse/OCPBUGSM-46114[*OCPBUGSM-46114*])

> * When you update configuration files that have the `musthave` compliance type, a new entry is created in the CR instead of overwriting the existing file. As a workaround, you can set the `complianceType` to `mustonlyhave` in the `PolicyGenTemplate` CR:

Reviewer 12: Hi. This line can be removed as the issue (appending instead updating) has been resolved starting starting 2.6.0+ and ported to ACM 2.5.2 and we no longer need to use the workaround. 

With the fix....policy should always update an object in an array if the `name` key matches what's in the policy. ACM fixes the issue [here](https://bugzilla.redhat.com/show_bug.cgi?id=2117728) and talks more about the fix in the comments. 

We are just waiting for a green light from QE before closing OCPBUGSM-41001.

* If an invalid subscription channel is specified in the subscription policy that is used to perform a cluster upgrade, the {cgu-operator-full} indicates a successful upgrade right after the policy is enforced because the `Subscription` state remains `AtLatestKnown`. (link:https://issues.redhat.com/browse/OCPBUGSM-43618[*OCPBUGSM-43618*])

* If the SRIOV-FEC Operator is installed through separate policies and the `CatalogSource` reuses the name of a default catalog source, the Operator installation might fail due to conflicts with the management of the default catalog sources. To avoid this issue the SRIOV-FEC `CatalogSource` CR should be added to the `common-config-policy` and the operator subscription should be added to the `common-subscriptions-policy`. As a workaround when using separate policies to install the SRIOV-FEC Operator, you must ensure that the `CatalogSource` for this Operator is uniquely named. (link:https://issues.redhat.com/browse/OCPBUGSM-39859[*OCPBUGSM-39859*])

> * The `SiteConfig` disk partition definition fails when applied to multiple nodes in a cluster. When a `SiteConfig` CR is used to provision a compact cluster, creating a valid `diskPartition` config on multiple nodes fails with a Kustomize plug-in error. (link:https://issues.redhat.com/browse/OCPBUGSM-44403[*OCPBUGSM-44403*])

Author: No, I don't think so. Just checking the bug: https://issues.redhat.com/browse/OCPBUGSM-44403 (see under actual results)
"Kustomize plug-in" is not the error message, but the summary of what the user would see.

. Get the authentication token by running the following command:
+
[source,terminal]
----
$ curl -i --insecure --request POST --header "OData-Version: 4.0" \
--header "Content-Type: application/json" -d '{"UserName": <BMC_USERNAME>, \
"Password": <BMC_PASSWORD>}' https://<BMC_IP>/redfish/v1/SessionService/Sessions/ |grep 'X-Auth-Token'
----
+
.Example output

[source,terminal]
----
X-Auth-Token: 1234abcd5678efgh9012ijkl3456mnop
----

. Using the authentication token, create the Redfish events subscription:
+
[source,terminal]
----
$ curl -X POST -i --insecure --header "X-Auth-Token: 1234abcd5678efgh9012ijkl3456mnop" \
-H 'Content-Type: application/json' --data-raw '{"Protocol": "Redfish", "Context": \
"Public", "Destination": "https://hw-event-proxy-openshift-hw-events.apps.example.com/webhook", \
"EventTypes": ["Alert"]}' https://<BMC_IP>/redfish/v1/EventService/Subscriptions
----

+

> You should receive a `201 Created` response and a header with `Location: https://<BMC_IP>/redfish/v1/EventService/Subscriptions/35` that indicates that the Redfish events subscription is successfully created. (link:https://issues.redhat.com/browse/OCPBUGSM-43707[*OCPBUGSM-43707*])

Author: I would prefer response here

----

.. Append the `rd.net.timeout.carrier=20` string to the `linux` boot line.

. Return to the initial working directory:
+
[source,terminal]
----
$ popd
----

. Generate the ISO file from the `iso-grub-cfg` directory:
+
[source,terminal]
----
$ mkisofs -JR -graft-points -o rhcos-carrier-timeout.iso iso-grub-cfg
----

. Push the updated ISO image to the server that is accessible by the hub cluster.

. On the hub cluster, update the `osImages` entry in the `AgentServiceConfig` CR for the installing release to point to the updated ISO image:
+
[source,terminal]
----
$ oc edit AgentServiceConfig agent
----

. Update the `url` field to point to the URL of the updated ISO image.

+
(link:https://issues.redhat.com/browse/OCPBUGSM-46336[*OCPBUGSM-46336*])

* A kernel failure occurs after rebooting a bare-metal SNO with DU profile and workload test application. As a workaround, you can add an additional kernel parameter to the performance profile:
+
[source,yaml]
----
apiVersion: performance.openshift.io/v2
kind: PerformanceProfile
spec:
  additionalKernelArgs:
  - rcutree.kthread_prio=11
----
+
(link:https://issues.redhat.com/browse/RHELPLAN-123262[*RHELPLAN-123262*])

> * During ZTP cluster deployments, the bare-metal host image provisioning might fail with an error referencing the HTTP 412 error code:

Reviewer 13: ```suggestion
* During ZTP cluster deployments, bare-metal host image provisioning might fail with an error referencing the HTTP 412 error code:
```

+
[source,terminal]
----

> Deploy step deploy.deploy failed with HTTPError: HTTP PATCH https://10.16.230.34/redfish/v1/Managers/1/VirtualMedia/EXT1 returned code 412. Base.1.8.GeneralError: A general error has occurred. See ExtendedInfo for more information. Extended information: [

Author: Added a closing bracket

{'MessageSeverity': 'Critical', 'MessageArgs': [], 'MessageId': 'Base.1.8.PreconditionFailed', 'Resolution': 'Try the operation again using the appropriate ETag.', '@odata.type': '#Message.v1_1_0.Message', 'Message': 'The ETag supplied did not match the ETag required to change this resource.'}
----

+

> This issue might affect various server models, in particular Gen8 HP machines and Gen9 HP machines running older firmware. For Gen9 HP machines, upgrading to the latest iLO firmware might resolve the issue. For Gen8 HP and others machines, there is no workaround for this issue. (link:https://issues.redhat.com/browse/OCPBUGS-1246[*OCPBUGS-1246*])

Reviewer 13: ```suggestion
This issue might affect various server models, in particular Gen8 HP machines and Gen9 HP machines running older firmware. For Gen9 HP machines, upgrading to the latest iLO firmware might resolve the issue. For Gen8 HP and others machines, currently there is no workaround for this issue. (link:https://issues.redhat.com/browse/OCPBUGS-1246[*OCPBUGS-1246*])
```

* On certain Lenovo models, for example the SE450, bare-metal host image provisioning during ZTP cluster deployments might fail with the HTTP 400 status code and a `PropertyNotWriteable` error:
+
[source,terminal]
----

> HTTP response for PATCH https://192.168.26.178/redfish/v1/Systems/1/Pending: status code: 400, error: Base.1.8.GeneralError: A general error has occurred. See ExtendedInfo for more information., extended: [{'MessageArgs': ['BootSourceOverrideEnabled'], 'Resolution': 'Remove the property from the request body and resubmit the request if the operation failed.', MessageId': 'Base.1.8.PropertyNotWritable', 'Message': 'The property BootSourceOverrideEnabled is a read only property and cannot be assigned a value.', '@odata.type': '#Message.v1_1_0.Message', 'MessageSeverity': 'Warning'}

Reviewer 14: First `[` does not back a closing bracket

Author: Added a closing bracket here as well

----

+
Currently, there is no workaround for this issue. (link:https://issues.redhat.com/browse/OCPBUGSM-46305[*OCPBUGSM-46305*])

* During a primary node replacement in a bare-metal cluster, the new primary host gets stuck in a `Provisioning` state, but the node reports into the cluster as `Ready`. (link:https://issues.redhat.com/browse/OCPBUGSM-45772[*OCPBUGSM-45772*])

* Using {rh-rhacm-first}, spoke cluster deployments on Dell PowerEdge R640 servers are blocked when the virtual media does not disconnect the ISO in the iDRAC console after writing the image to the disk. To workaround this issue, disconnect the ISO manually through the Virtual Media tab in the iDRAC console. (link:https://issues.redhat.com/browse/OCPBUGSM-45884[*OCPBUGSM-45884*])

> * On a dual-stack networking environment, devices and connections get stuck in an `ip-check` state with a default `dhcp6` profile generated by the `nm-initrd-generator` utility. The `/etc/resolve.conf` file is not generated. To workaround this issue, restart the `NetworkManager` service. This generates the missing `/etc/resolve.conf` file and the installation can continue. (link:https://issues.redhat.com/browse/RHELPLAN-127788[*RHELPLAN-127788*], link:https://issues.redhat.com/browse/OCPBUGS-70[*OCPBUGS-70*])

Reviewer 14: ```suggestion
* On a dual-stack networking environment, devices and connections get stuck in an `ip-check` state with a default `dhcp6` profile generated by the `nm-initrd-generator` utility. Due to this issue, the `/etc/resolve.conf` file is not generated. To workaround this issue, restart the `NetworkManager` service. This generates the missing `/etc/resolve.conf` file and the installation can continue. (link:https://issues.redhat.com/browse/RHELPLAN-127788[*RHELPLAN-127788*], link:https://issues.redhat.com/browse/OCPBUGS-70[*OCPBUGS-70*])
```

> * Using NVIDIA-branded Mellanox NICs on Dell hardware, incoming packets that are larger than the preset F5 application receive buffer (currently set to 8K) arrive with an incorrect VLAN tag. This results in sporadic truncated packets being delivered by the NIC. (link:https://issues.redhat.com/browse/RHELPLAN-123058[*RHELPLAN-123058*])

Reviewer 14: `Using NVIDIA-branded` -> `When using NVIDIA-branded`
and maybe 
`sporadic` -> `irregular` 
Up to you though!

* Sometimes, a {sno} node configured with static IP and deployed using the GitOps ZTP pipeline becomes unreachable during Day 2 operator configuration. The {product-title} cluster installation completes successfully, and the cluster is healthy. After reboot, the node becomes unreachable with the network interface down. (link:https://issues.redhat.com/browse/OCPBUGSM-46688[*OCPBUGSM-46688*])

* After removing the `SriovNetworkNodePolicy` policy from the Git repository, the `SriovNetworkNodePolicy` resource managed by the removed policy remains on the spoke cluster. (link:https://issues.redhat.com/browse/OCPBUGSM-34614[*OCPBUGSM-34614*])

> * When upgrading the PTP Operator from 4.10 to 4.11, the OpenShift subscription reports the error 'no channel heads (entries not replaced by another entry) found in channel `"stable"` of package `"ptp-operator"`. However, the Operator upgrades successfully. (link:https://issues.redhat.com/browse/OCPBUGSM-46114[*OCPBUGSM-46114*])

Reviewer 14: The 
'no channel heads (entries not replaced by another entry) found in channel `"stable"` of package `"ptp-operator"`
has a single quote opening and no closing punctuation.

> * Currently, all `PtpConfig` CRs in the ZTP source CRs includes the `phc2sysOpts` option. Thus, even if the user does not include `phc2sysOpts` in the user `PolicyGenTemplate` CR, the `phc2sysOpts`option is added to the spoke PTP configuration. If PTP with dual NIC is configured through ZTP, the user needs to update one `PtpConfig` CR to remove the `phc2sysOpts` option after ZTP is completed. (link:https://issues.redhat.com/browse/OCPBUGSM-47798[*OCPBUGSM-47798*])

Reviewer 14: ```suggestion
* Currently, all `PtpConfig` CRs in the ZTP source CRs includes the `phc2sysOpts` option. Thus, even if the user does not include `phc2sysOpts` in the user `PolicyGenTemplate` CR, the `phc2sysOpts` option is added to the spoke PTP configuration. If PTP with dual NIC is configured through ZTP, the user needs to update one `PtpConfig` CR to remove the `phc2sysOpts` option after ZTP is completed. (link:https://issues.redhat.com/browse/OCPBUGSM-47798[*OCPBUGSM-47798*])
```

Reviewer 14: This whole sentence renders in back tics without that space. This is how it looks in the preview: 

the ``phc2sysOpts`option is added to the spoke PTP configuration. If PTP with dual NIC is configured through ZTP, the user needs to update one `PtpConfig`` CR to remove.

Author: Adding the space resolves this

* When secure boot is enabled `stalld` service fails to start because it is not able to open the `/sys/kernel/debug/sched_features` file. (link:https://issues.redhat.com/browse/OCPBUGS-1466[*OCPBUGSM-1466*])

> * When secure boot is enabled, the `kdump` service might fail to start with `kexec: failed to load kdump kernel` error. To work around this issue, add `efi=runtime`to the kernel arguments. (link:https://issues.redhat.com/browse/OCPBUGS-97[*OCPBUGSM-97*])

Reviewer 14: ```suggestion
* When secure boot is enabled, the `kdump` service might fail to start with `kexec: failed to load kdump kernel` error. To workaround this issue, add `efi=runtime` to the kernel arguments. (link:https://issues.redhat.com/browse/OCPBUGS-97[*OCPBUGSM-97*])
```

. On the hub cluster, update the `osImages` entry in the `AgentServiceConfig` CR for the installing release to point to the updated ISO image:
+
[source,terminal]
----
$ oc edit AgentServiceConfig agent
----

. Update the `url` field to point to the URL of the updated ISO image.

+
(link:https://issues.redhat.com/browse/OCPBUGSM-46336[*OCPBUGSM-46336*])

* A kernel crash occurs after rebooting a bare-metal SNO with DU profile and workload test application. As a workaround, you can add an additional kernel parameter to the performance profile:
+
[source,yaml]
----
apiVersion: performance.openshift.io/v2
kind: PerformanceProfile
spec:
  additionalKernelArgs:
  - rcutree.kthread_prio=11
----
+
(link:https://issues.redhat.com/browse/RHELPLAN-123262[*RHELPLAN-123262*])

* During ZTP cluster deployments, the bare-metal host image provisioning fails with the following error: `The ETag supplied did not match the ETag required to change this resource`. To workaround this issue, attach the ISO image to the host using the Baseboard Management Controller GUI. (link:https://issues.redhat.com/browse/OCPBUGSM-46305[*OCPBUGSM-46305*])

> * During a master node replacement in a bare-metal cluster, the new master host gets stuck in a `Provisioning` state, but the node reports in to the cluster as `Ready`. (link:https://issues.redhat.com/browse/OCPBUGSM-45772[*OCPBUGSM-45772*])

Reviewer 15: LGTM
